selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/theia/TheiaBuildPluginTest.java

```diff
@@ -125,7 +123,7 @@ public void pluginShouldBeBuilt() {
     }
   }
 
-  @Test(priority = 1)
+  @Test(priority = 1, groups = OPENSHIFT)
```

Comment:
If each public method in test class should be run on _OpenShift_ only, we can simplify the code by placing `@Test(groups = OPENSHIFT)` above the class declaration, and removing `groups = OPENSHIFT` from the test methods.